docs/CHANGELOG.md

```diff
@@ -43,6 +43,8 @@ If minConfirmations > 0 and failOnRevert=true then the ethtx task will error on
 
 If `minConfirmations` is not set on the task, the chain default will be used which is usually 12 and always greater than 0.
 
+- `http` task now allows specification of request headers. Use like so: `foo [type=http headers="[\\"X-Header-1\\", \\"value1\\", \\"X-Header-2\\", \\"value2\\"]"]`.
```

Comment:
Did you consider other formats, like the native colon-separated key-value pairs?

Reply:
```suggestion
- `http` task now allows specification of request headers. Use like so: `foo [type=http headers="[\\"X-Header-1: value1\\", \\"X-Header-2: value2\\", \\"Set-Cookie: UserID=JohnDoe; Max-Age=3600; Version=1\\"]"]`.
```
I'm cursious how nested quoted strings will look too: e.g. from wikipedia `Alt-Svc: http/1.1="http2.example.com:8001"; ma=7200`